protected/tests/unit/models/AUSPatientTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: root
+ * Date: 22/09/17
+ * Time: 2:11 PM
+ */
+
+class AUS_PatientTest extends CDbTestCase
+{
+    protected $fixtures = array('patient' => 'Patient');
+    public function setUp()
+    {
+        parent::setUp();
+    }
+
+    public function tearDown()
+    {
+        parent::tearDown();
+    }
+
+    public function testUniqueMedicareNo()
+    {
+        //create a patient as a duplicate of one in the database
+        $patient_dup = new Patient;
+        $patient_dup->attributes = $this->patient['patient1'];
+
+        //change the things we don't want it to fail on
+        $patient_dup->id = NULL;
+        $patient_dup->pas_key = NULL;
+        $patient_dup->hos_num = 65465465;
+
+        //fail to save with duplication
+        $this->assertFalse($patient_dup->save());
```

Comment:
You might want to check exactly which field the save failed on. $patient_dup->getError('nhs_num') could be useful